web/public/mgmt/config.yml

```diff
@@ -2133,6 +2133,10 @@ collections:
               - { label: Error - No Elevator Registrations Found, name: errorTextNoElevatorRegistrations, widget: string }
               - { label: Error - Missing Required Fields, name: errorTextFieldsRequired, widget: string }
               - { label: Error - Search Failed, name: errorTextSearchFailed, widget: string }
+              - { label: Informational Message - In Review, name: informationalInReview, widget: string }
```

Comment:
Have you added the following fields to the search.tsx file>

Reply:
I've added a refactor ticket to do this and add the preview